target/linux/ar71xx/image/legacy-devices.mk

```diff
@@ -427,6 +427,12 @@ define LegacyDevice/dLAN_pro_1200_ac
 endef
 LEGACY_DEVICES += dLAN_pro_1200_ac
 
+define LegacyDevice/DW33D
+  DEVICE_TITLE := DomyWifi DW33D
+  DEVICE_PACKAGES := kmod-usb-core kmod-usb2 kmod-usb-storage kmod-ledtrig-usbdev kmod-ath10k
+endef
+LEGACY_DEVICES += DW33D
```

Comment:
@mkresin It's fixed.  But build the image size is only 4.7M.  
Error message：The uploaded image file does not contain a supported format.